packages/terra-date-time-picker/src/_TimeClarification.module.scss

```diff
@@ -66,3 +66,4 @@
     display: none;
   }
 }
+/* stylelint-enable */
```

Comment:
Can we re-enable the specific rules instead of stylelint as a whole? It's clearer what's being disabled and re-enabled.

Reply:
For sure, I'll update later on tonight

Reply:
![watch](https://user-images.githubusercontent.com/6720991/51772539-d8328300-20b1-11e9-82a1-19f1bf62519f.gif)

Reply:
ope, got caught up in a buncha stuff 😬